src/modules/mavlink/CMakeLists.txt

```diff
@@ -80,3 +76,18 @@ px4_add_module(
 if(PX4_TESTING)
 	add_subdirectory(mavlink_tests)
 endif()
+
+if (NOT ${PX4_PLATFORM} STREQUAL "nuttx" OR CONFIG_BUILD_FLAT)
```

Comment:
These could be replaced with directly publishing sensor_accel, sensor_gyro, etc.

Reply:
Yes, good point. I even have some implementation ready for this, but left it out in fear of breaking something. Was going to make another PR of it

Reply:
Do you want the generic, pure uORB based, implementation into this PR? Or should I make another PR before this one? Or would it be ok to continue with another PR later?

Reply:
@dagar I only now realized that you have already cleaned up the px4 driver libs to be pure uORB publishers. I just removed this change, and removed the linking to drivers__device from src/lib/drivers/accelerometer and src/lib/drivers/gyroscope CMakeLists.txt.